README.md

```diff
@@ -134,56 +134,58 @@ cdk deploy --all
+| CREATE_MLSPACE_CLOUDTRAIL_TRAIL                |                                                                                                                                                                               Whether or not to create an MLSpace trail within the account                                                                                                                                                                                |                              `true` |
+| NEW_USERS_SUSPENDED                            |                                                                                                                                                                     Whether or not new user accounts will be created in a suspended state by default                                                                                                                                                                      |                             `false` |
+| ENABLE_TRANSLATE                               |                                                                                                                        Whether or not translate capabilities will be deployed/enabled in MLSpace. If translate is not available in the region you are deploying to you should set this to `false`                                                                                                                         |                              `true` |
+| LAMBDA_RUNTIME                                 |                                                                                                                             The lambda runtime to use for MLSpace lambda functions and layers. This needs to be a python runtime available in the region in which MLSpace is being deployed.                                                                                                                              |                         Python 3.11 |
+| LAMBDA_ARCHITECTURE                            |                                                                           The architecture on which to deploy the MLSpace lambda functions. All lambda layers will also need to be built for the selected archiecture. You can do this by ensuring you run the `cdk deploy` command from a machine with the same architecture you're targeting.                                                                           |                                 x86 |
```

Comment:
What is this change for? Best I can tell you added a couple parameters but the formatting of everything seems jacked now. Maybe it is just how github is showing it?

Reply:
I added the new tables `GROUPS_TABLE_NAME` and `GROUP_USERS_TABLE_NAME` to this, seems my IntelliJ reformatted the tables so that they are now the same length for every column.